core/command.h

```diff
@@ -98,6 +99,18 @@ int main (int cmdline_argc, char** cmdline_argv)
     ::MR::GUI::App app (cmdline_argc, cmdline_argv);
 #endif
     ::MR::App::parse ();
+
+    //ENVVAR name: MRTRIX_CLI_PARSE_ONLY
+    //ENVVAR Set the command to parse the provided inputs and then quit
+    //ENVVAR if it is 1. This can be used in the CI of wrapping code,
+    //ENVVAR such as the automatically generated Pydra interfaces
+    char* parse_only = std::getenv("MRTRIX_CLI_PARSE_ONLY");
+    if (parse_only && ::MR::to<int>(parse_only)) {
+      WARN(
+        "Quitting after parsing command-line arguments successfully because environment variable 'MRTRIX_CLI_PARSE_ONLY' is set to '1'"
+      );
```

Comment:
```suggestion
      INFO(
        "Quitting after parsing command-line arguments successfully due to environment variable 'MRTRIX_CLI_PARSE_ONLY'"
      );
```

-   Warning-level is for when there is some kind of ambiguity or uncertainty regarding how to proceed, that the user needs to be alerted to, but the command is going to attempt to proceed nevertheless.

-   Environment variable value shouldn't necessarily need to be "`1`".